terms/experimentalData/organRIN.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "sage.annotations-experimentalData.organRIN-0.0.1",
+    "type": "number",
+    "description": "Organ RIN, as previously measured on organ or region"
```

Comment:
I'm not clear on what this means

Reply:
@karawoo If I change it to "Organ RNA Integrity Number, as previously measured on organ or region", would that clarify it?

Reply:
It's the "as previously measured on organ or region" that I don't really understand. "previously" makes it sound like maybe it has changed or become out of date? and is "region" here brain region -- if so we might want to clarify that.

Reply:
I talked to @amapeters, and she said that it's asking if there is any RIN information that has been previously measured on the tissue or brain region and that it's a measure of the quality of the tissue the specimen came from that may indicate other issues.  She is going to look at how it was explained in other consortia.